orchestration/scripts/inject-file-ids.sh

```diff
@@ -23,7 +23,7 @@ declare -ra BQ_QUERY=(
   FROM ${TABLE} S LEFT JOIN \`${JADE_PROJECT}.${JADE_DATASET}.datarepo_load_history\` J
   ON J.state = 'succeeded'
   AND JSON_EXTRACT_SCALAR(S.descriptor, '$.crc32c') = J.checksum_crc32c
-  AND '${GCS_DATA_DIR}' || JSON_EXTRACT_SCALAR(S.descriptor, '$.file_name') = J.source_name"
+  AND '/' || JSON_EXTRACT_SCALAR(S.descriptor, '$.file_id') || '/' || JSON_EXTRACT_SCALAR(S.descriptor, '$.file_name') = J.target_path"
```

Comment:
idle thought:  would it be more readable to use `CONCAT()` here instead of `||`?